packages/core/src/saml-applications/routes/index.ts

```diff
@@ -162,17 +168,18 @@ export default function samlApplicationRoutes<T extends ManagementApiRouter>(
     '/saml-applications/:id/secrets',
     koaGuard({
       params: z.object({ id: z.string() }),
-      body: z.object({ lifeSpanInDays: z.number().optional() }),
+      // The life span of the SAML app secret is in years (at least 1 year), and for security concern, secrets which never expire are not recommended.
+      body: z.object({ lifeSpanInYears: z.number().int().gte(1) }),
```

Comment:
Seems like a breaking change?

Reply:
Yes, considering that our API is currently in the development phase and has not been released yet, breaking changes are acceptable. Additionally, it is more reasonable to use years rather than days as the unit for the lifespan of a secret/certificate.